### PR TITLE
change the storage impl

### PR DESCRIPTION
**Pull request by ChrisEddy**

### CODE REVIEW

File: rig.js [hunk 1]
```diff
@@ -60,26 +60,30 @@ class RigManager {
...
-    this.localRig = new Avatar(o, {
-      fingers: true,
-      hair: true,
-      visemes: true,
-      // decapitate: selectedTool === 'firstperson',
-    });
-    this.scene.add(this.localRig.model);
-
-    await this.localRigQueue.unlock();
+    try {
```
Comment (avaer) [inline rig.js:63]: Why try?

File: rig.js [hunk 2]
```diff
@@ -100,28 +104,32 @@ class RigManager {
...
-      fingers: true,
-      hair: true,
-      visemes: true,
-      // decapitate: selectedTool === 'firstperson',
-    });
-    this.scene.add(peerRig.model);
-    this.peerRigs.set(peerId, peerRig);
-
-    await this.peerRigQueue.unlock();
+    try {
```
Comment (avaer) [inline rig.js:107]: Why try? If this is meant to catch fetch errors, it should be scoped tighter.

File: rig.js (hunk 1 above)
Comment (ChrisEddy) [reply rig.js:63]: URL can bounce and was crashing site, uncaught promise rejection

File: rig.js (hunk 2 above)
Comment (ChrisEddy) [reply rig.js:107]: URL can bounce and was crashing site, uncaught promise rejection

File: rig.js (hunk 1 above)
Comment (avaer) [reply rig.js:63]: Then we should scope this down to the actual problematic fetch, rather than all errors we might see randomly.

**Comment by ChrisEddy** [comment]
@avaer fixed

**Review by avaer — APPROVED** [review]
Looks good!

**Pull request merged**